micropython/test_micropython.c

```diff
@@ -0,0 +1,180 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Used for MicroPython testing. Runs MicroPython script.
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Piotr Nieciecki
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+
+#define MICROPYTHON_BIN "/bin/micropython "
+#define PATH_TO_TESTS   "/usr/test/micropython/"
+
+#define CMDLINE_PREFIX     "# cmdline: "
+#define CMDLINE_PREFIX_LEN strlen(CMDLINE_PREFIX)
```

Comment:
```suggestion
#define CMDLINE_PREFIX_LEN (sizeof(CMDLINE_PREFIX) - 1)
```
As we know the CMDLINE_PREFIX at the compile time there is no need to call strlen